src/utils/handle-errors.js

```diff
@@ -32,7 +33,7 @@ function formatUnhandledRejectionReason (reason) {
     if (reason instanceof Error)
         return reason.stack;
 
-    return Object.prototype.toString.call(reason);
+    return util.inspect(reason, { depth: 1 });
```

Comment:
I think we should discuss the depth limit with other team members. I think it would be useful to get maximum information when an error is raised, so we shouldn't limit the depth here. Or maybe make it configurable.